cmd/homi/docker/doc.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2018 The klaytn Authors
+// Copyright 2016 The go-ethereum Authors
+// This file is part of go-ethereum.
+//
+// Copyright 2017 AMIS Technologies
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+//
+
+/*
+docker package contains compose and service packages which are to be used to create a docker-compose.yml with given options
```

Comment:
'contains compose' is strange.
```suggestion
docker package contains service packages which are to be used to create a docker-compose.yml with given options
```